drivers/ISOSR.py

```diff
@@ -613,15 +665,15 @@ def create(self, sr_uuid, size):
             smconfig['iso_type'] = self.dconf['type']
             self.session.xenapi.SR.set_sm_config(self.sr_ref, smconfig)
 
-        # CA-80254: Check for iso/img files whose name consists of extended
-        # characters.
-        for f in util.listdir(self.path, quiet=True):
-            if is_image_utf8_compatible(f):
-                try:
-                    f.encode('ascii')
-                except UnicodeEncodeError:
-                    raise xs_errors.XenError('CIFSExtendedCharsNotSupported',
-                            opterr='The repository contains at least one file whose name consists of extended characters.')
+        _, num_images_ignored = list_images(self.path)
+        if num_images_ignored > 0:
+            xapi = self.session.xenapi
+            xapi.message.create("DISK_IMAGES_IGNORED",
+                                "4",
+                                "SR",
+                                self.uuid,
+                                "Ignored disk image file(s) due to"
```

Comment:
Maybe this should include the `num_images_ignored` in the message?

Reply:
I was going to do that, but I didn't know how well that would play with message localization - given that XenCenter would presumably have to parse the message and then recreate a localized version of it.

Reply:
XC only supports English now, localization was removed

Reply:
Regardless. Messages should be internationalized.

Reply:
I wouldn't worry so much about the message text itself because the UI can always provide an override for it, but if we want the UI to show the number too, it would be useful to include it in the message in a form that the UI can parse it easily. For example, there are some messages with more complex info/more parameters that include the message body as xml - but this may be too much here. A simpler idea would be to use this message: `N ignored disk image file(s) due to file name encoding issues`, so the UI expects that the first word of this message is always the number, so it can fish it out easily.